Makefile

```diff
@@ -28,6 +28,13 @@ update:
 	go get -t -v -d -u ./...
 	@echo "Dependencies updated"
 
+.PHONY: update
+update-schema:
+	go build -v -tags 'libsqlite3 devel' ./lxd/schema.go
+	./schema
+	rm schema
```

Comment:
Why not use "go run" rather than build a temporary binary and discard it afterwards?